src/src/rx-serial/SerialHoTT_TLM.cpp

```diff
@@ -37,6 +38,27 @@ int SerialHoTT_TLM::getMaxSerialReadSize()
     return HOTT_MAX_BUF_LEN - hottInputBuffer.size();
 }
 
+void SerialHoTT_TLM::setTXMode()
+{
+#if defined(PLATFORM_ESP32)
+    // assumes half duplex line is connected to Serial RX GPIO
```

Comment:
Minor but normally tx pins are used for half duplex e.g. stm and flight controllers. It might be a little more natural to use txPin?

Reply:
What are the odds. Ah, 50%. 

RX is on ch13 (leftmost) for clean looks. Other than that I have no excuse. I'll do it and retest the whole thing.

Reply:
Changed to use TX.

Reply:
Sorry to our testers who already have rx connected, but this will be better long term.